_data/simple-icons.json

```
@@ -2315,7 +2315,7 @@
         },
         {
             "title": "CodeSandbox",
-            "hex": "000000",
+            "hex": "151515",
             "source": "https://codesandbox.io"
```

```suggestion
            "source": "https://codesandbox.io/CodeSandbox-Press-Kit.zip"
```

Let's use the link of the press kit here.